### DISCUSSION TITLE
GitHub Student Pack not being bound to the account

### DISCUSSION THREAD

**Discussion by forgerlil** [community/community#36002] [GitHub Education]
Hello!

I have been having an issue with attaching my GitHub Student Pack with my account. I see other issues don't include people with links provided to them by a partner (If I missed it please point it out to me).

I am a student of an affiliated partner, who provided me with a Student Pack link, hence I don't need to provide ID cards or other sorts of proof as student. 

Whenever I click on the student pack generated link provided by my school, I get redirected to this page (attempting to sign in here instead of clicking on the main button brings me to <code>https://education.github.com/</code>) :
![link-welcome](https://user-images.githubusercontent.com/81853712/195552199-0918746e-bdc8-45d9-bf4f-47a937b34d5e.png)

Clicking on the main button takes me to this page(<code>https://education.github.com/pack/join</code>): 
![are-you-student](https://user-images.githubusercontent.com/81853712/195552234-0f86208b-6d09-4ee0-bb8d-5e15aba31f6f.png)

Clicking on "Yes, I'm a student" brings me to the main page(<code>https://education.github.com/</code>):
![education-root](https://user-images.githubusercontent.com/81853712/195552265-8a798d54-418e-4cae-8cf7-3add976c7ff8.png)

However, whenever I try to access my offers at <code>https://education.github.com/pack#offers</code> (because <code>https://education.github.com/pack/offers</code> redirects me to <code>https://education.github.com/pack/join</code>), I still get prompted to sign up for the pack (which I know from fellow students that the button would not be there were the pack bound to my account properly) :
![gh-pack-prompt](https://user-images.githubusercontent.com/81853712/195552307-bcec2950-49ff-4b66-8e9e-4bf1ed9dae1d.png)

And therefore the offers naturally fail to have any redirection links for me to access them, since the Pack is not applied to my account adequately:
![no-links](https://user-images.githubusercontent.com/81853712/195552338-c2e90a99-63e6-4ba3-ab33-45beca56b495.png)

Furthermore, I often get stuck in a click loop, for example the <code>Benefits</code> button on the main page just refreshes the page, and most other links redirect me to the "Are you a student?" page, which is quickly getting frustrating.

Attempting to reverify my academic affiliation is to no avail, I get asked to contact my school:
![affiliation](https://user-images.githubusercontent.com/81853712/195552362-7eb98363-1f92-4301-91b4-60cd1d0fbaae.png)

I have checked with my school if any steps I did were wrong, but all seems to be in order from mine and the school's side. Could you please point out the solution or see whether there's something not working as intended?

Thanks :)

**Comment by GarayevF** [comment]
Any updates?

**Comment by Ronit11246** [comment]
Hi, I've been having the same issue. Any updates

**Reply by GarayevF** [reply]
i opened link in incognito tab and it worked
